src/engraving/libmscore/range.cpp

```diff
@@ -92,6 +92,13 @@ void TrackList::appendTuplet(Tuplet* srcTuplet, Tuplet* dstTuplet)
             Tuplet* st = toTuplet(de);
             Tuplet* dt = toTuplet(e);
             appendTuplet(st, dt);
+        } else if (Segment* seg = toSegment(de->explicitParent())) {
```

Comment:
Before you cast this object to a Segment, you must check if it really is a Segment 

```
EngravingItem* parent = de->explicitParent();
...
} else if (parent->isSegment()) {
    Segment* seg = toSegment(parent);
...
}
```